### PR TITLE
Update 03_installing_efm.mdx

### PR DESCRIPTION
**Pull request by moiznalwalla**
Fixed a typo and renamed a link to the destination page name.

## What Changed?

Please list, at a high level, what changed in your branch. If you changed content, include the product, guide, and version as applicable.

**Examples**

- Fixed typo in EPAS 13 epas_inst_linux guide
- Added more detail to ARK 3.5 getting started guide introduction
- Fixed broken link on `/supported-open-source/pgbackrest/05-retention_policy/` page

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

**Comment by moiznalwalla** [comment]
Fixing doc bug as descrived in https://enterprisedb.atlassian.net/browse/DF-264

**Comment by drothery-edb** [comment]
The second issue in DF-264 is a broken link on step 4 on the docs/efm/latest/efm_user/12_upgrading_existing_cluster topic. Are you going to address that in a separate commit? If so, I can approve and merge the PR with both issues addressed and update the ticket accordingly.

**Comment by moiznalwalla** [comment]
The second issue: The link works fine only had a name that would not make sense to the user (like section 5), so I renamed it to the destination page. 

https://613a4840763452651457801c--edb-docs-staging.netlify.app/docs/efm/latest/efm_user/03_installing_efm/

Let me know if something is still missing.

File: product_docs/docs/efm/4/efm_user/12_upgrading_existing_cluster.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ The efm `upgrade-conf` utility locates the `.properties` and `.nodes` files of p
 !!! Note
     `db.bin` is a required property. When modifying the properties file, ensure that the `db.bin` property specifies the location of the Postgres `bin` directory.
 
-4.  If you are using Eager Failover, you must disable it before stopping the EFM cluster. For more information, see [Disabling Eager Failover](06_configuring_for_eager_failover/disabling_the_eager_failover).
+4.  If you are using Eager Failover, you must disable it before stopping the EFM cluster. For more information, see [Disabling Eager Failover](06_configuring_for_eager_failover/#disabling-eager-failover).
```
Comment (drothery-edb) [inline product_docs/docs/efm/4/efm_user/12_upgrading_existing_cluster.mdx:28]:
```suggestion
4.  If you are using Eager Failover, you must disable it before stopping the EFM cluster. For more information, see [Disabling Eager Failover](04_configuring_efm/06_configuring_for_eager_failover/#disabling-eager-failover).
```

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**